app/preprints/detail/controller.ts

```diff
@@ -152,6 +152,10 @@ export default class PrePrintsDetailController extends Controller {
         return false;
     }
 
+    get showOSFBanner(): boolean {
+        return  this.model.provider.id === 'osf';
```

Comment:
Nit: I don't think this is needed, as there may be some other criteria in the future to show this banner, but there is also the `config.defaultProvider` that may be appropriate to use here. We use it in other places to basically check the same thing, but it feels marginally more robust to be using a config rather than a hardcoded providerID